config/backend-config/single_workspace.go

```diff
@@ -22,6 +22,7 @@ type singleWorkspaceConfig struct {
 	configBackendURL *url.URL
 	configJSONPath   string
 	configEnvHandler types.ConfigEnvI
+	cpRouterURL      string
```

Comment:
This doesn't seem to be needed. How come you added it?

Reply:
Wanted to remain consistent across the three modules but removed it now